RCS file: /cvs/gsl/fpicker/source/unx/gnome/SalGtkFilePicker.cxx,v
retrieving revision 1.5
diff -u -p -u -r1.5 SalGtkFilePicker.cxx
--- fpicker/source/unx/gnome/SalGtkFilePicker.cxx	18 Mar 2005 09:48:46 -0000	1.5
+++ fpicker/source/unx/gnome/SalGtkFilePicker.cxx	11 Apr 2005 13:28:32 -0000
@@ -302,11 +306,11 @@ SalGtkFilePicker::SalGtkFilePicker( cons
 	gtk_file_chooser_set_extra_widget( GTK_FILE_CHOOSER( m_pDialog ), m_pVBox );
 
 	g_signal_connect( G_OBJECT( m_pToggles[PREVIEW] ), "toggled",
 					  G_CALLBACK( preview_toggled_cb ), this );
-//	g_signal_connect( G_OBJECT( m_pFilterComboBox ), "changed",
-//					  G_CALLBACK( filter_changed_atsave_cb ), this );
+	g_signal_connect( G_OBJECT( m_pFilterComboBox ), "changed",
+					  G_CALLBACK( filter_changed_atsave_cb ), this );
 	g_signal_connect( G_OBJECT( m_pDialog ), "notify::filter",
 					  G_CALLBACK( filter_changed_cb ), this );
 
     gtk_widget_show( m_pVBox );
 }
